tests/test_fov.py

```diff
@@ -0,0 +1,127 @@
+"""Unit tests for fov.py."""
+
+import tempfile
+
+import numpy as np
+
+from simSPI import fov
+
+pdb_test_string = (
+    "ATOM  46616  CA  LYS C  71      79.058 -46.138  59.506  1.00 30.00      IB   C\n"
+    "ATOM    150  CA  ASN A  21     -26.864  63.622 -59.845  1.00 30.00      A    C\n"
+    "ATOM    172  CA  VAL A  24     -20.237  55.210 -60.973  1.00 30.00      A    C"
+)
```

Comment:
yes, very good! just inline a snippet of data. be aware the pdb data is very silly and all the spaces between columns is uber critical. that's one reason why it was deprecated. so be sure to copy and paste EXACTLY from a real pdb file to avoid column offset